cmd/payments/main.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/provider"
+	"storj.io/storj/pkg/transport"
+)
+
+var (
+	ctx = context.Background()
+	// ErrPaymentsDial throws when there are errors dialing the payments client
+	ErrPaymentsDial = errs.Class("error dialing payments client")
+
+	// ErrRequest is for gRPC request errors after dialing
+	ErrRequest = errs.Class("error processing request")
+
+	// ErrIdentity is for errors during identity creation for this CLI
+	ErrIdentity = errs.Class("error creating identity")
+
+	// ErrArgs throws when there are errors with CLI args
+	ErrArgs = errs.Class("error with CLI args")
+
+	port string
+
+	rootCmd = &cobra.Command{Use: "payments"}
+
+	cmdGenerate = &cobra.Command{
+		Use:   "generatecsv",
+		Short: "generates payment csv",
+		Args:  cobra.MinimumNArgs(2),
+		RunE:  generatecsv,
+	}
+
+	cmdTest = &cobra.Command{
+		Use:   "test",
+		Short: "test only: add records to accounting rollup",
+		RunE:  test,
+	}
+)
+
+// Payments gives access to the payments api
+type Payments struct {
+	client pb.PaymentsClient
+}
+
+func main() {
+	rootCmd.PersistentFlags().StringVarP(&port, "port", "p", ":10000", "storj-sdk satellite port")
```

Comment:
It's going to be storj-sim, now, I think :D

Reply:
awesome ty